enginetest/queries/query_plans.go

```diff
@@ -10257,12 +10263,13 @@ WHERE
 	{
 		Query: `
 	SELECT COUNT(*) FROM NOXN3`,
-		ExpectedPlan: "GroupBy\n" +
-			" ├─ select: COUNT(*)\n" +
-			" ├─ group: \n" +
-			" └─ Table\n" +
-			"     ├─ name: NOXN3\n" +
-			"     └─ columns: [id brqp2 fftbj a7xo2 kbo7r ecdkm numk2 letoe ykssu fhcyt]\n" +
+		ExpectedPlan: "Project\n" +
+			" ├─ columns: [COUNT(1):0!null as COUNT(*)]\n" +
+			" └─ GroupBy\n" +
+			"     ├─ select: COUNT(1 (bigint))\n" +
+			"     ├─ group: \n" +
+			"     └─ Table\n" +
+			"         └─ name: NOXN3\n" +
```

Comment:
no table projection means reading all rows in this case?

Reply:
No, that's actually means it's projecting 0 columns, which is dumb. I want to fix that but it's just a lot bigger.

Reply:
how does it count the rows? does it just count a bunch of `sql.Row{nil}`s?